### PR TITLE
Add Laravel workflows

### PR DESCRIPTION
**Pull request by robmellett**
## Description of changes (updated or new workflows)

Added some workflows for using Laravel.

I'm not sure who to attribute the author fields to - so give me a shout if I've labeled them incorrect.

image

### CODE REVIEW

File: specs/laravel/laravel_create_new_project.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Create a new Laravel App using Composer
+command: composer create-project laravel/laravel example-app
+tags:
+  - composer
+  - laravel
+  - php
+description: ~
```
Comment (alokedesai) [inline specs/laravel/laravel_create_new_project.yml:8]: Are there any relevant descriptions here that would make it clearer what this command does?

**Review by alokedesai — COMMENTED** [review]
Change looks great! Just see my comment about whether a description would help here

File: specs/laravel/laravel_create_new_project.yml (hunk 1 above)
Comment (robmellett) [reply specs/laravel/laravel_create_new_project.yml:8]: Yeap no worries, just updated the command descriptions

Comment (robmellett) [reply specs/laravel/laravel_create_new_project.yml:8]: Thanks for the review btw <3

**Review by alokedesai — APPROVED** [review]
LGTM! Thank you for contributing!

**Pull request merged**